src/middleware/http_logger.py

```
@@ -1,35 +1,38 @@
+from starlette.middleware.base import BaseHTTPMiddleware
+
+from src.util import get_user, request_id_var
 
 http_logger = logging.getLogger("http_access")
 
+
 class HTTPLoggerMiddleware(BaseHTTPMiddleware):
     async def dispatch(self, request: Request, call_next):
         try:
             user = get_user(request)
         except:
```

Ruff (0.13.3)

Do not use bare `except`

(E722)